__tests__/endpoints.test.js

```diff
@@ -42,4 +41,39 @@ describe('GET/api', ()=>{
             })
         })
     })
-})
\ No newline at end of file
+});
+
+describe('GET/api/articles/:article_id', ()=>{
+    test('returns a 200 status code and the requested article object to the client', ()=>{
+        return request(app)
+        .get('/api/articles/2')
+        .expect(200)
+        .then(({body})=>{
+            const article= body.article
+            expect(article.title).toBe('Sony Vaio; or, The Laptop')
+            expect(article.topic).toBe('mitch')
+            expect(article.author).toBe('icellusedkars')
+            expect(typeof article.body).toBe('string')
+            expect(typeof article.created_at).toBe('string')
+            expect(typeof article.votes).toBe('number')
+            expect(article.article_img_url).toBe('https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700')
+        })
```

Comment:
For a more flexible test suite - I would suggest looking at the .toMatchObject or .objectContaining matchers :)